### PR TITLE
FEMA Restful service urls

### PR DESCRIPTION
**Pull request by fernando-aristizabal**
Related to: https://github.com/hyriver/pygeohydro/pull/111

- Added new FEMA Restful service urls.
- Changed `pygeoogc.ServiceURL.restful.fema` to `fema_nfhl`.
- updated history and authors.

 - [X] Changes and the contributor name are documented in `HISTORY.rst`.

### CODE REVIEW

File: AUTHORS.rst [hunk 1]
```diff
@@ -11,3 +11,4 @@ Contributors
 ------------
 
 * `Tim Cera <https://github.com/timcera>`__: :pull_ogc:`58`
+* `Fernando Aristizabal <https://github.com/fernando-aristizabal>`_
```
Comment (cheginit) [inline AUTHORS.rst:14]: The RST format for URL is \`text \<url\>\`__ (note the two underscores). Please add the missing underscore.

File: HISTORY.rst [hunk 2]
```diff
@@ -2,6 +2,22 @@
 History
 =======
 
+0.15.2 (2023-XX-XX)
+-------------------
+Contributor: [Fernando Aristizabal](https://github.com/fernando-aristizabal)
```
Comment (cheginit) [inline HISTORY.rst:7]: Please fix the link following the RST format that I mentioned in the previous comment.

File: HISTORY.rst [hunk 3]
```diff
@@ -2,6 +2,22 @@
...
+if you have ``pygeoogc==0.14.x`` installed, you cannot install
+``pygeoogc==0.15.x`` series. This is to ensure that the API is
+consistent across all minor versions.
+
+New Features
+~~~~~~~~~~~~
+- Added RESTfulURLs for FEMA's National Flood Hazard Layer (NFHL) service.
+
+0.15.1 (2023-07-27)
 0.15.2 (2023-0X-XX)
```
Comment (cheginit) [inline HISTORY.rst:21]: Please remove this line.

File: AUTHORS.rst (hunk 1 above)
Comment (cheginit) [reply AUTHORS.rst:14]: Also, please reference the PR number like the previous contributor: :pull_ogc:\`62\`

**Review by cheginit — COMMENTED** [review]
I added a few minor comments.

**Comment by cheginit** [comment]
Great, thank again for your PR!

**Comment by fernando-aristizabal** [comment]
My apologies but I had to fix a few of the URLs. I'm still trying to find a way to test with nox when the dependencies are within the `install_deps` function within the noxfile.

Is there a way to reopen the PR on your end so I can push the latest to the same PR. Otherwise, I'll open another one.

**Comment by cheginit** [comment]
Sure, no problem. You can open another PR. Do you need a new dependency?

**Comment by fernando-aristizabal** [comment]
I'll open up a new PR.

My difficulty is that running nox tests within pygeohydro uses the latest commit of `hyriver/pygeoogc@main`. My proposed changes are typically under `fernando-aristizabal/pygeoogc@nfhl`.

**Comment by cheginit** [comment]
Ah, ok. I see what you mean. I think the easiest way is to just merge the new `pygeoogc` PR and then continue with the `pygeohydro` test using `nox`.

**Pull request merged**